src/domain/collection/curatedEventList/CuratedEvents.tsx

```diff
@@ -0,0 +1,135 @@
+import { Button } from 'hds-react';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import LoadingSpinner from '../../../common/components/spinner/LoadingSpinner';
+import { CollectionFieldsFragment } from '../../../generated/graphql';
+import Container from '../../app/layout/Container';
+import { getEventIdsFromUrls } from '../../event/EventUtils';
+import { EventType } from '../../event/types';
+import styles from './curatedEventList.module.scss';
+import EventCards from './EventCards';
+import OnlyExpiredEvents from './OnlyExpiredEvents';
+import { usePaginatedEventsByIdsQuery } from './utils';
+
+const PAST_EVENTS_DEFAULT_SIZE = 4;
+export const PAGE_SIZE = 10;
+
+export const coursesListTestId = 'curated-courses-list';
+export const eventsListTestId = 'curated-events-list';
+
+interface Props {
+  collection: CollectionFieldsFragment;
+}
+
+const CuratedEvents: React.FC<Props> = ({ collection }) => {
+  const { t } = useTranslation();
+  const { eventIds, courseIds } = React.useMemo(
+    () => getEventIdsFromUrls(collection.curatedEvents),
+    [collection.curatedEvents]
+  );
+
+  return (
+    <Container className={styles.container}>
+      {!!eventIds.length && (
+        <CollectionEventsList
+          testId={eventsListTestId}
+          eventIds={eventIds}
+          eventType="event"
+          title={t('collection.curatedEvents.eventsTitle')}
+        />
+      )}
+      {!!courseIds.length && (
```

Comment:
It seems that all the courses are always at the end of the collection, but in cms they can be in mixed order:
<img width="580" alt="Screenshot 2021-03-26 at 1 54 58 PM" src="https://user-images.githubusercontent.com/7648194/112628246-4cf0db80-8e3b-11eb-8ad2-fe2cd78b8d61.png">
Is the collection designed to show them in two groups? This might be confusing to cms producer that the order has changed in the collection page...

Reply:
Idea is to show the in their own groups.